servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -323,7 +323,9 @@ public StreamingHttpClient buildStreaming() {
             }
 
             final LoadBalancer<LoadBalancedStreamingHttpConnection> lb =
-                    closeOnException.prepend(ctx.builder.loadBalancerFactory.newLoadBalancer(sdEvents,
+                    closeOnException.prepend(ctx.builder.loadBalancerFactory.newLoadBalancer(
+                            ctx.address().toString(),
```

Comment:
1. How users can override this value if they are not satisfied with what we pass automatically?
2. For a scenario when users use the same proxy to connect to any address (think of 100s different targets), the value returned from `ctx.address()` will be useless as it always indicates a proxy. Consider writing a method that will mix `ctx.proxyAddress` and `ctx.builder.address` together. It will provide a full picture of what is the target resource and how we connect (directly or via proxy).

Reply:
1. Fair point. They can't. I'm not sure we need to provide that flexibility at this point however.
2. Thank you for pointing that out. I just added a utility to combine the two.

Having both points in mind - I'm not necessarily in favor of over-engineering this feature at this time. It's a simple string at this moment, meant for debugging purposes. If we need users to have more flexibility, we could consider more API changes, but my intention was to bring useful functionality with absolutely minimum level of inconvenience. Most users should not notice the change. I'm open to suggestions though - perhaps we can do something without too many changes? I can't see it at the moment, but if there are ways - let's go for it.

Reply:
+1, let's not overcomplicate API now. If necessary, users can wrap `LoadBalancerFactory` and then `LoadBalancer` interface to intercept the passed `targetResource` and change it.